python/postprocessing/framework/postprocessor.py

```diff
@@ -20,6 +22,12 @@ def __init__(self,outputDir,inputFiles,cut=None,branchsel=None,modules=[],compre
 	self.noOut=noOut
 	self.friend=friend
 	self.justcount=justcount
+	self.provenance=provenance
+	self.jobReport = JobReport() if fwkJobReport else None
+	self.haddFileName=haddFileName
+	if self.jobReport and not self.haddFileName :
+		print "Because you requested a FJR we assume you want the final hadd. No name specified for the output file, will use tree.root"
+		self.haddFileName="tree.root"
```

Comment:
io forse qui metterei throw

Reply:
actually tree.root is the name the crab config expects, so it is fine to have a (this) default